src/views/Kitten.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="kitten-box-container">
+    <img
+      :src="nameOfFile.src"
+      alt="nameOfFile.description"
+      class="picture-style"
+    />
+    <div class="picture-description">{{ nameOfFile.description }}</div>
+    <!-- <ul>
+      <li>Tags: {{ nameOfFile.tags }}</li>
+      <li>Date: {{ nameOfFile.date }}</li>
+    </ul> -->
+
+    <div class="button-pictures">
+      <button
+        @click="previous"
+        class="button-images-each"
+        variant="outline-primary"
+        href="#"
+      >
+        Previous
+      </button>
+      <button
+        @click="next"
+        class="button-images-each"
+        variant="outline-primary"
+        href="#"
+      >
+        Next
+      </button>
+      <p>{{ nameOfFile.tags }}</p>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.kitten-box-container {
+  border: 8px solid #6134c6;
```

Comment:
Is there a better way to put a colour in with scss?

Reply:
Hi Thomas!
I could use the variables defined in the style file. 
I didn't see that particular purple tho. 😀

Reply:
Yes we updated the colour in our branding but it hasn't been updated in this repo yet 😊 
But you can always add another variable yourself. Best to avoid hard coding colours like this

Reply:
Perfect!! Will do that!